README.md

```diff
@@ -1,18 +1,33 @@
+# dfeshiny <a href="https://dfe-analytical-services.github.io/dfeshiny/"><img src="man/figures/logo.png" align="right" height="120" alt="dfeshiny website" /></a>
+
 <!-- badges: start -->
 [![R-CMD-check](https://github.com/dfe-analytical-services/dfeshiny/actions/workflows/R-CMD-check.yaml/badge.svg)](https://github.com/dfe-analytical-services/dfeshiny/actions/workflows/R-CMD-check.yaml)
 [![Codecov test coverage](https://codecov.io/gh/dfe-analytical-services/dfeshiny/branch/main/graph/badge.svg)](https://app.codecov.io/gh/dfe-analytical-services/dfeshiny?branch=main)
+[![Lifecycle: experimental](https://img.shields.io/badge/lifecycle-experimental-orange.svg)](https://lifecycle.r-lib.org/articles/stages.html#experimental)
 <!-- badges: end -->
 
-<a href="http://dfe-analytical-services.github.io/dfeshiny/"><img src="man/figures/dfeshiny.png" align="right" width="120" /></a>
-
-# dfeshiny  
-
 An R package to support analysts in developing official DfE dashboards and help 
 them meet the necessary standards required of public facing government services.
 
-## Installing the package
+## Installation
 
-To install, run `renv::install("dfe-analytical-services/dfeshiny")`.
+dfeshiny is not currently available on CRAN. For the time being you can
+install the development version from GitHub.
+
+If you are using
+[renv](https://rstudio.github.io/renv/articles/renv.html) in your
+project (recommended):
+
+``` r
+renv::install("dfe-analytical-services/dfeR")
```

Comment:
Update to dfeshiny

Reply:
Impressed with my past self for failing to notice I was telling people to install the wrong package. Fixed!